CHANGELOG.md

```diff
@@ -7,6 +7,20 @@ Unreleased
 ### SDK Bugs
 
 
+Release v1.8.0 (2022-09-13)
+================================
+### SDK Enhancements
+* Replace error type assertions with errors.As in DefaultFormattingStrategy [#PR 353](https://github.com/aws/aws-xray-sdk-go/pull/353)
+* Dummy segments don't need cancel go routine [#PR 365](https://github.com/aws/aws-xray-sdk-go/pull/365)
```

Comment:
For each change, if it would be considered a PATCH change by the definitions of https://semver.org/, it should be considered a bug fix. For example, I think this would probably be considered a backwards-compatible bug fix rather than adding functionality